tests/upload.js

```diff
@@ -165,6 +165,25 @@ describe("File upload", function () {
 
         imagekit.upload(fileOptions);
     });
+   
+    it('Buffer file larger than 10MB', function (done) {
+        const fileOptions = {
+            fileName: "test_file_name",
+            file: fs.readFileSync(path.join(__dirname,"./data/test_video_larger_than_10mb.mp4"))
```

Comment:
Instead of putting a video file in the SDK repo, let's programmatically generate a 10+ MB file for testing purpose.

You can use the `Buffer.alloc()` method for this. I used this snippet to generate a 13 MB long base64 string:

```
Buffer.alloc(10000000).toString('base64')
```

Let me know if you have questions.

https://nodejs.org/api/buffer.html#static-method-bufferallocsize-fill-encoding